packages/imperative/src/plugins/cmd/validate/validate.definition.ts

```diff
@@ -44,5 +62,9 @@ export const validateDefinition: ICommandDefinition = {
       description: "Validate all installed plug-ins",
       options    : ""
     },
+    {
+      description: "Validate a plug-in named my-plugin, and treat warnings as errors",
+      options    : "my-plugin --fow"
```

Comment:
Shouldn't the example always use the full name for options instead of aliases?

Reply:
I don't know how consistent we are with that rule - https://github.com/zowe/zowe-cli/blob/master/packages/cli/src/zostso/start/address-space/AddressSpace.definition.ts

But, I will change it. 😃

Reply:
Thanks for changing it.
Maybe we should open an issue about this one too

Reply:
https://github.com/zowe/imperative/issues/637